packages/desktop-client/src/components/reports/disabledList.ts

```diff
@@ -53,59 +55,59 @@ const totalGraphOptions: graphOptions[] = [
   {
     description: 'TableGraph',
     disabledSplit: [],
-    defaultSplit: 'Category',
+    defaultSplit: t('Category'),
     disabledType: [],
-    defaultType: 'Payment',
+    defaultType: t('Payment'),
     disableLegend: true,
     disableLabel: true,
   },
   {
     description: 'BarGraph',
     disabledSplit: [],
-    defaultSplit: 'Category',
+    defaultSplit: t('Category'),
     disabledType: [],
-    defaultType: 'Payment',
+    defaultType: t('Payment'),
   },
   {
     description: 'AreaGraph',
-    disabledSplit: ['Category', 'Group', 'Payee', 'Account'],
-    defaultSplit: 'Interval',
+    disabledSplit: [t('Category'), t('Group'), t('Payee'), t('Account')],
+    defaultSplit: t('Interval'),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should the strings in this file be translated?
cc @carkom

Reply:
I think these are saved in the DB too (but not 100% sure hence the tag of carkom). 

If they are saved in the DB - the values must remain in english, but we can change the labels shown to users (might require further code changes).

Reply:
Ah, I see. I'll wait with this then.

Reply:
Same as above. They are saved to the DB in English and need to stay that way in order to properly re-create the saved report.